.circleci/scripts/should_run_job.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -ex -o pipefail
```

Comment:
Can you add -u for no reads from unassigned bash variables? You'll need to change -z "$CIRCLE_PULL_REQUEST" to -z "${CIRCLE_PULL_REQUEST:-}"